src/main/java/de/sakpaas/backend/service/OccupancyService.java

```diff
@@ -36,25 +47,19 @@ public OccupancyService(OccupancyRepository occupancyRepository) {
    * @param time        the time to calculate with
    * @return the average occupancy
    */
-  public static Double calculateAverage(List<Occupancy> occupancies, ZonedDateTime time) {
+  private Double calculateAverage(List<Occupancy> occupancies, ZonedDateTime time) {
```

Comment:
Ich würde die Funktion umbennen in: claculateWeightedOccupancy, da sie ja auf keinen Fall einen Durchschnitt berechnet.

Reply:
Sehe ich ein, aber unser Model heißt `AccumulatedOccupancy` dann nehme ich das